### PR TITLE
Yacht details page 

### PR DESCRIPTION
**Pull request by akos29**
Yacht details page 
--
In  this request majorly:

- [x] Add feature to show a details page when the user selects a specific item, they can see the [details page](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579) with its full description
- [x] Add feature to enable user to click the "Reserve" button to book a yacht.

You can find [the link for API PR](https://github.com/akos29/book_appointment/pull/26/)

This will resolve issue https://github.com/akos29/book_appointment/issues/8

### CODE REVIEW

**Review by mahabubx7 — CHANGES_REQUESTED** [review]
## 🛑 STATUS: REQUIRED CHANGES 🛑 

gif

Please recover the `tailwindcss` packages & setup. Follow up the settings from `display` branch #5 

- [ ] Also make sure the API (backend) is providing the details instead of static data.
- [ ] Tailwindcss should work properly

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+dist
```
Comment (mahabubx7) [inline .gitignore:2]: **[OPTIONAL]**
It's really good that you added `dist` in the ignore, but we are using `CRA` or create-react-app. So, we also need to add `build` folder in here.

File: src/App.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/extensions */
```
Comment (mahabubx7) [inline src/App.js]: You should not use `eslint-disable` at all. Try to fix or reach out your team-members to solve the issues.

File: src/components/Display.jsx [hunk 3]
```diff
@@ -18,14 +21,20 @@ function Display() {
         <h1>Loading ...</h1>
       ) : (
         <>
-          <h1>Display yatchs!</h1>
+          <h1>Display yachts!</h1>
```
Comment (mahabubx7) [inline src/components/Display.jsx]: **[OPTIONAL]**
The word `yatch` is incorrect. We all fixed that to `yacht`. You can do it too.

File: src/components/yacht/YachtDetail.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    user_id: 1,
+    yacht_id: 1,
+    created_at: '2023-08-20T08:00:00Z',
+    updated_at: '2023-08-20T08:00:00Z',
+    image_path: '/path/to/image1.jpg',
+    yacht_name: 'Luxury Cruiser',
+    price: 1500,
+    duration: '4 hours',
+    yacht_model: 'Model X',
+  };
```
Comment (mahabubx7) [inline src/components/yacht/YachtDetail.jsx]: Please try to resolve or make the API call (backend-repo) first and then implement in frontend app. So, that we can use redux store and actual data from API without using fake or static data.

File: src/index.css [hunk 5]
```diff
@@ -25,7 +25,3 @@ code {
     'Courier New',
     monospace;
 }
-
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
```
Comment (mahabubx7) [inline src/index.css]: Please, recover this `tailwindcss` set-up. it is necessary for those who used it in this project.

**Review by lugard1 — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @akos29  , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done installing the missing packages. ✔
- It's good you resolved the typo issue as requested by the previous reviewer.✔
- The requirements have been met with success. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by lugard1 — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @akos29  , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- Fix merge issues. ✔
- You documented your work professionally. ✔
- The requirements have been met with success. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**